pages/linux/screenkey.md

```diff
@@ -0,0 +1,32 @@
+# screenkey
+
+> A screencast tool to display your keys.
+> More information: <https://www.thregr.org/~wavexx/software/screenkey/>.
+
+- Display keys which are currently being pressed on the screen:
+
+`screenkey`
+
+- Display keys and mouse buttons which are currently being pressed on the screen:
+
+`screenkey --mouse`
+
+- Launch the settings menu of screenkey:
+
+`screenkey --show-settings`
+
+- Change the position of screenkey:
```

Comment:
Does it open a new window at this position or does it change the position of a running screenkey instance?

Reply:
@marchersimon It creates a new instance and places it on the given position. If there is a running instance in the background, running `screenkey --position top` will create a new screenkey instance at specified position i.e. there would be two instances running simultaneously.

Reply:
Then how about:
```suggestion
- Launch screenkey at a specific position:
```

Reply:
How about the above change @CleanMachine1 @bl-ue ?

Reply:
How about the above change @CleanMachine1 @bl-ue ?

Reply:
Please dont ping us twice.

Reply:
It was an accident, I believe @CleanMachine1 :)

Reply:
Looks good to me @ashishbinu!

Reply:
> Please dont ping us twice.

Sorry I didn't know it was sent twice.My internet was kinda slow when I tried to send it.

Reply:
Ah, ok, just sent 2 requests. No harm! Just seemed like you were spamming a little :+1: